helm-chart/renku/templates/data-service/rbac_k8s_watcher.yaml

```diff
@@ -0,0 +1,78 @@
+{{- if .Values.dataService.rbac.create -}}
```

Comment:
question: Should we remove the corresponding stuff from the notebook service / golang k8s watcher? I guess we can do that in a followup pr if you want.